### ISSUE TITLE
Добавить «Как работает position: sticky в CSS»

### ISSUE THREAD

**Issue by pepelsbey** [web-standards-ru/new#100]
Добавить MD-файл в [content/articles](https://github.com/web-standards-ru/new/tree/master/content/articles)

- [Как работает position: sticky в CSS](https://medium.com/p/bc7ff7088693)
- [mediumexporter](https://github.com/xdamman/mediumexporter)

В начале файла добавить мету:

```
---
title: "TITLE"
date: "YYYY.MM.DD"
---
```

**Comment by JediYozh** [comment]
Я сделаю, только картинку или картинки? Там их несколько.

**Comment by pepelsbey** [comment]
Да, можно начать с картинок. Потом можно докинуть Маркдаун, это просто, на самом деле.

**Comment by JediYozh** [comment]
Да это понятно...просто в статье несколько картинок, а в описании issue ты говоришь про одну, вот и спросил имел ли ты в виду все :)

**Comment by pepelsbey** [comment]
Поправил :)

**Comment by JediYozh** [comment]
#103
